doc/scripts/extract_content.py

```diff
@@ -54,6 +56,10 @@ def get_rst_files(dest, dir):
 
         copy_if_different(file, os.path.join(dest, frel))
 
+        # Inspect only .rst files
```

Comment:
Would be work adding a comment about why we're inspecting .rst files:

    # Inspect only .rst files for directives referencing other files
    # we'll need to copy (as configured in the DIRECTIVES variable)